frontend/src/Css.tsx

```diff
@@ -40,12 +40,15 @@ export const color = {
   successWeak: '#e6f4ea', // Google green 50
   terminated: '#80868b',
   theme: '#1a73e8',
-  themeDarker: '#0b59dc',
+  themeDarker: '#0049b5',
   warningBg: '#f9f9e1',
   warningText: '#ee8100',
   infoBg: '#f3f4ff',
   infoText: '#1a73e8',
   weak: '#9aa0a6',
+  link: '#0d47a1',
+  linkLight: '#5472d3',
+  whiteSmoke: '#f3f3f3',
```

Comment:
where does the name "whiteSmoke" come from?

Reply:
This is currently used as background color during hover for table view, but the RGB color code was defined in individual component. So I extract them to Css.tsx file here.